packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -1488,7 +1489,8 @@ class DropdownButtonFormField<T> extends FormField<T> {
          onSaved: onSaved,
          initialValue: value,
          validator: validator,
-         autovalidate: autovalidate,
+         // ignore: deprecated_member_use_from_same_package
```

Comment:
lets remove this for now

Reply:
here and other places

Reply:
You mean remove the autovalidate parameter? But what if someone used the FormField Widget directly?

Reply:
I mean the `// ignore: deprecated_member_use_from_same_package` and reintroduce it in the deprecation PR later

Reply:
Hahah. Right, I misunderstood you